.classpath

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="android/src"/>
+	<classpathentry kind="src" path="android/build/.apt_generated"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Library/Android/sdk/platforms/android-26/android.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Library/Application Support/Titanium/mobilesdk/osx/8.0.0.GA/android/titanium.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Library/Application Support/Titanium/mobilesdk/osx/8.0.0.GA/android/kroll-common.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Library/Application Support/Titanium/mobilesdk/osx/8.0.0.GA/android/kroll-apt.jar"/>
+	<classpathentry kind="src" path=".apt_generated">
+		<attributes>
+			<attribute name="optional" value="true"/>
+		</attributes>
+	</classpathentry>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/lib/firebase-auth-16.2.1.aar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/lib/firebase-auth-interop-16.0.1.aar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/build/intermediates/exploded-aar/firebase-auth-16.2.1/classes.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/build/intermediates/exploded-aar/firebase-auth-interop-16.0.1/classes.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/build/intermediates/exploded-aar/play-services-tasks-16.0.1/classes.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/build/intermediates/exploded-aar/firebase-common-16.1.0/classes.jar"/>
+	<classpathentry kind="lib" path="/Users/ravindra/Documents/github/titanium-firebase-auth/android/lib/firebase-common-16.1.0.aar"/>
+	<classpathentry kind="output" path="bin"/>
+</classpath>
```

Comment:
We can remove these

Reply:
yes, added it to the gitignore and removed the file